localgovdrupal/localgov_base
# Fixes #366 - text overlapping Guide Nav

## What does this change?



Fixes https://github.com/localgovdrupal/localgov_base/issues/366

It adds a `word-wrap: break-word` rule to  `.lgd-guide-nav__list` class, just like GitHub does for its comment panels 
![image](https://github.com/user-attachments/assets/e5bd8958-c971-477d-b4e4-824088f1e9f7)

  
## How to test



With On a site using `localgov_base` theme with some Guide Pages created, and at least Guide Page with a very long title with no spaces:


1. Visit a Guide Overview page that is the parent for those Guide pages
2. See the Guide Nav section
3. **Before PR changes:** Verify that the Guide Page with long title overflows the container
4. **Before PR changes:** Verify that the Guide Page with long title does not overflow the container

## How can we measure success?



1. less reports of web URLs or email addresses overflowing on smaller mobile phones
2. no new issues that are worse that come up due to adding this text breaking rule.

## Have we considered potential risks?



There were concerns in this comment by @andybroomfield about the impact of similar rules applied in general, and there was a suggestion to apply only to emails on smaller devices. But, those risks were not familiar to me, so I don't see risk, as overflowing the container on any screen size will look worse than a non-ideal wrapping in my opinion.

## Images



**BEFORE**
![Screenshot 2024-09-25 at 23 09 45](https://github.com/user-attachments/assets/67d64f71-da93-457d-88b2-4371bb20af49)

**AFTER**
![Screenshot 2024-09-25 at 23 08 21](https://github.com/user-attachments/assets/f1ea4622-3659-4a84-9b1f-649f17785537)


## Accessibility
